dev/src/bulk-writer.ts

```diff
@@ -285,9 +285,10 @@ class BulkCommitBatch extends WriteBatch {
         );
         this.pendingOps[i].onSuccess(new WriteResult(updateTime));
       } else {
-        const error = new (require('google-gax').GoogleError)(
-          status.message || undefined
-        );
+        const error =
+          new (require('google-gax/build/src/fallback').GoogleError)(
```

Comment:
I'm wondering if instead of using this path to fallback, we can use a subpath export to the fallback? https://nodejs.org/api/packages.html#subpath-exports

Maybe it's already set up that way, I didn't look at gax source yet. But the path indicates maybe it is not.

Reply:
That's a great idea (and "today I learned" as well).

We will be able to safely start using it when we drop Node 12 support. The documentation says subpath exports were added in v12.7.0 but we have `"engines.node"` set to `">=12"` now, so we cannot possibly start using it without making a semver major. But for the next major I definitely want to do it. I filed https://github.com/googleapis/gax-nodejs/issues/1337 so that we don't forget.